app/renderer/components/Session/Session.js

```diff
@@ -16,6 +16,8 @@ const {TabPane} = Tabs;
 
 const ADD_CLOUD_PROVIDER = 'addCloudProvider';
 
+const HEIGHT_OF_SERVICE_CONFIG_AREA = 400;
```

Comment:
I'd rather define the constant in a separate file to avoid possible cross references